diff --git a/node_modules/bfx-api-node-rest/lib/.rest2.js.swp b/node_modules/bfx-api-node-rest/lib/.rest2.js.swp
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/bfx-api-node-rest/lib/rest2.js b/node_modules/bfx-api-node-rest/lib/rest2.js
index 0a09a6e..0f8b493 100644
--- a/node_modules/bfx-api-node-rest/lib/rest2.js
+++ b/node_modules/bfx-api-node-rest/lib/rest2.js
@@ -54,7 +54,7 @@ const API_URL = 'https://api.bitfinex.com'
  */
 function _takeResNotify (data) {
   const notification = new Notification(data)
-  return notification
+  return notification.notifyInfo
 }
 
 /**
@@ -382,8 +382,8 @@ class RESTv2 {
    * @returns {Promise} p
    * @see https://docs.bitfinex.com/v2/reference#rest-public-books
    */
-  orderBook (symbol = 'tBTCUSD', prec = 'P0', cb) {
-    return this._makePublicRequest(`/book/${symbol}/${prec}`, cb)
+  orderBook (symbol = 'tBTCUSD', prec = 'P0', len = 25, cb) {
+    return this._makePublicRequest(`/book/${symbol}/${prec}?len=${len}`, cb)
   }
 
   /**
@@ -532,9 +532,10 @@ class RESTv2 {
     timeframe = '1m',
     symbol = 'tBTCUSD',
     section = 'hist',
+    period = null,
     query = {}
   }, cb) {
-    let url = `/candles/trade:${timeframe}:${symbol}/${section}`
+    let url = period ? `/candles/trade:${timeframe}:${symbol}:${period}/${section}` : `/candles/trade:${timeframe}:${symbol}/${section}`
 
     if (Object.keys(query).length > 0) {
       url += `?${new URLSearchParams(query).toString()}`
@@ -1279,6 +1280,10 @@ class RESTv2 {
    * @returns {Promise} p
    */
   submitOrder (order, cb) {
+    // bfx-api-node-models@1.8.2+ wraps as { order: <Order> }
+    if (order && order.order && typeof order.order.toNewOrderPacket === 'function') {
+      order = order.order
+    }
     const packet = order.toNewOrderPacket()
 
     if (this._affCode) {
@@ -1293,8 +1298,8 @@ class RESTv2 {
       .then((res) => {
         // 2 orders can be returned if OCO was used. But due to the interface
         // we can only return one. User should use getOrders instead
-        const orders = _takeResNotify(res).notifyInfo || []
-        return orders[0] || []
+        const orders = _takeResNotify(res) || []
+        return orders || []
       })
   }
 
@@ -1499,6 +1504,10 @@ class RESTv2 {
    * @returns {Promise} p
    */
   submitFundingOffer (offer, cb) {
+    // bfx-api-node-models@1.8.2+ wraps as { offer: <FundingOffer> }
+    if (offer && offer.offer && typeof offer.offer.toNewOfferPacket === 'function') {
+      offer = offer.offer
+    }
     const packet = offer.toNewOfferPacket()
 
     if (this._affCode) {
