diff --git a/node_modules/ffmpeg/lib/ffmpeg.js b/node_modules/ffmpeg/lib/ffmpeg.js
index e5e99e9..d698fac 100644
--- a/node_modules/ffmpeg/lib/ffmpeg.js
+++ b/node_modules/ffmpeg/lib/ffmpeg.js
@@ -17,7 +17,7 @@ var ffmpeg = function (/* inputFilepath, settings, callback */) {
 		// Instance the new arrays for the format
 		var format = { modules : new Array(), encode : new Array(), decode : new Array() };
 		// Make the call to retrieve information about the ffmpeg
-		utils.exec(['ffmpeg','-formats','2>&1'], settings, function (error, stdout, stderr) {
+		utils.exec(['ffmpeg','-i',`"${fileInput}"`,'2>&1'], settings, function (error, stdout, stderr) {
 			// Get the list of modules
 			var configuration = /configuration:(.*)/.exec(stdout);
 			// Check if exists the configuration
@@ -62,7 +62,7 @@ var ffmpeg = function (/* inputFilepath, settings, callback */) {
 		// New 'promise' instance 
 		var deferred = when.defer();
 		// Make the call to retrieve information about the ffmpeg
-		utils.exec(['ffmpeg','-i',fileInput,'2>&1'], settings, function (error, stdout, stderr) {
+		utils.exec(['ffmpeg','-i',`"${fileInput}"`,'2>&1'], settings, function (error, stdout, stderr) {
 			// Perse output for retrieve the file info
 			var filename		= /from \'(.*)\'/.exec(stdout) || []
 			  , title			= /(INAM|title)\s+:\s(.+)/.exec(stdout) || []
